test/games_collection_test.rb

```diff
@@ -49,7 +49,7 @@ def test_it_can_count_game_by_season
       "20132014" => 6
     }
 
-    assert_equal expected, @games_collection.count_of_games_by_season
+    assert_equal expected, @games_collection.count_of_games_per_season
   end
 
   def test_it_grabs_highest_total_score
```

Comment:
The method name that Iteration 2 calls for is `average_goals_by_season` - if you want to discard/revert the changes on this file, my "quick fix" pull request updates it to the way the spec harness expects.